graylog2-server/src/main/java/org/graylog2/indexer/searches/SearchesClusterConfig.java

```diff
@@ -44,6 +46,19 @@
             .put(Period.days(30), "Search in the last 30 days")
             .put(Period.ZERO, "Search in all messages")
             .build();
+    private static final Map<Period, String> DEFAULT_SURROUNDING_TIMERANGE_OPTIONS = ImmutableMap.<Period, String>builder()
+            .put(Period.seconds(1), "1 second")
+            .put(Period.seconds(5), "5 seconds")
+            .put(Period.seconds(10), "10 seconds")
+            .put(Period.minutes(1), "1 minute")
```

Comment:
I think we need at least one option between 10 seconds and 1 minute, the gap seems to big to me. How about adding 30 seconds by default?